server.go

```diff
@@ -981,11 +984,25 @@ func (s *Server) HandleMetricPacket(packet []byte, protocolType ProtocolType) er
 			samples.Add(ssf.Count("packet.error_total", 1, map[string]string{"packet_type": "metric", "reason": "parse"}))
 			return err
 		}
-		s.Workers[metric.Digest%uint32(len(s.Workers))].PacketChan <- *metric
+		s.IngestMetric(metric)
 	}
 	return nil
 }
 
+func (s *Server) IngestMetric(metric *samplers.UDPMetric) {
+	if metric.Digest == 0 {
```

Comment:
The underlying assumption here is that for metrics that come in through the OpenMetrics source, `metric.Digest` is no longer guaranteed to be non-empty. We could add a comment here explaining this, or refactor the code so that this is self documenting.

Reply:
Added a comment.